creator-node/src/services/stateMachineManager/index.js

```diff
@@ -53,15 +53,36 @@ class StateMachineManager {
 
     // Upon completion, make queue jobs record metrics and enqueue other jobs as necessary
     const queueNameToQueueMap = {
-      [QUEUE_NAMES.MONITOR_STATE]: monitorStateQueue,
-      [QUEUE_NAMES.FIND_SYNC_REQUESTS]: findSyncRequestsQueue,
-      [QUEUE_NAMES.FIND_REPLICA_SET_UPDATES]: findReplicaSetUpdatesQueue,
-      [QUEUE_NAMES.MANUAL_SYNC]: manualSyncQueue,
-      [QUEUE_NAMES.RECURRING_SYNC]: recurringSyncQueue,
-      [QUEUE_NAMES.UPDATE_REPLICA_SET]: updateReplicaSetQueue,
-      [QUEUE_NAMES.RECOVER_ORPHANED_DATA]: recoverOrphanedDataQueue
+      [QUEUE_NAMES.MONITOR_STATE]: {
+        queue: monitorStateQueue,
+        maxWaitingJobs: 10
+      },
+      [QUEUE_NAMES.FIND_SYNC_REQUESTS]: {
+        queue: findSyncRequestsQueue,
+        maxWaitingJobs: 10
+      },
+      [QUEUE_NAMES.FIND_REPLICA_SET_UPDATES]: {
+        queue: findReplicaSetUpdatesQueue,
+        maxWaitingJobs: 10
+      },
+      [QUEUE_NAMES.MANUAL_SYNC]: {
+        queue: manualSyncQueue,
+        maxWaitingJobs: 1000
```

Comment:
not ideal to limit manual sync queue since thats what syncs content on upload, but tbh if that queue ever has 1000 waiting jobs we have much bigger problems

fine for now i guess